.github/workflows/ci.yaml

```diff
@@ -248,10 +238,24 @@ jobs:
         #   CARGO_TARGET_X86_64_UNKNOWN_LINUX_MUSL_LINKER: ${{ matrix.cc }}
 
       - name: Build - Node
+        if: matrix.target != 'x86_64-unknown-linux-musl'
         working-directory: node-bindings
         run: |
           npm install --ignore-scripts
-          npm run build-${{ matrix.platform }}-${{ matrix.architecture }}
+          npm run build-${{ env.SHORT_TARGET_NAME }}
+
+      - name: Build - Node (linux-musl)
+        if: matrix.target == 'x86_64-unknown-linux-musl'
+        uses: docker://rust:alpine3.15
+        env:
+          RUSTFLAGS: -C target-feature=-crt-static
+        with:
+          entrypoint: /bin/sh
+          args: -c "
+            cd node-bindings &&
+            apk add alpine-sdk nodejs npm git &&
+            npm install --ignore-scripts &&
+            npm run build-${{ env.SHORT_TARGET_NAME }}"
```

Comment:
@CharlieC3 this looks appalling, but was the only approach I could get working. Let me know if you have any suggestions on how to improve this.

Reply:
Curious what happens if you use the `cross` tool to compile on the `musl` target in the normal `Build - Node` step? [It would look something like this](https://gist.github.com/Thearas/8a656c2e3783ec1a623146f672e21ee3).

Reply:
That tool is wrapper for a script pretty similar to the above, it wraps some docker commands. In this case, it wouldn't really help much because we'd still need run scripts within cross's docker image to install nodejs. 

Also, I don't think it would work even after that, because their image is still doing cross-compilation from an ubuntu base image (https://github.com/cross-rs/cross/blob/master/docker/Dockerfile.x86_64-unknown-linux-musl) using musl-cross tooling, which runs into various problems outputting the type of library we're building for nodejs integration. The only way I was able to get it to work was building from a musl environment (like alpine).